README.md

```diff
@@ -72,16 +72,33 @@ And do not forget to enable XML output for the Maven plugin:
 </build>
 ```
 
+Please make sure that by default workflows on `pull_request` events checkout [`refs/pull/:prNumber/merge`](https://help.github.com/en/actions/reference/events-that-trigger-workflows) instead of the head of the pull request.
```

Comment:
I think you wanted to say something like "Please note that..." instead of "Please make sure that..". Because "Please make sure" means "Please verify"

Reply:
I'll change the phrasing in the readme 😄